src/PrestaShopBundle/Bridge/Smarty/HeaderConfigurator.php

```diff
@@ -0,0 +1,390 @@
+        $this->translator = $translator;
+        $this->hookDispatcher = $hookDispatcher;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * Assign smarty variables for the header.
+     *
+     * @param ControllerConfiguration $controllerConfiguration
+     *
+     * @return void
+     */
+    public function configure(ControllerConfiguration $controllerConfiguration): void
+    {
+        $controllerConfiguration->templatesVars['table'] = $controllerConfiguration->table;
+        $controllerConfiguration->templatesVars['current'] = $this->router->generate('admin_features_index');
+        $controllerConfiguration->templatesVars['token'] = $controllerConfiguration->token;
+        $controllerConfiguration->templatesVars['host_mode'] = (int) defined('_PS_HOST_MODE_');
+        $controllerConfiguration->templatesVars['stock_management'] = (int) $this->configuration->get('PS_STOCK_MANAGEMENT');
+        $controllerConfiguration->templatesVars['no_order_tip'] = $this->getNotificationTip('order');
+        $controllerConfiguration->templatesVars['no_customer_tip'] = $this->getNotificationTip('customer');
+        $controllerConfiguration->templatesVars['no_customer_message_tip'] = $this->getNotificationTip('customer_message');
+
+        if ($controllerConfiguration->displayHeader) {
+            $controllerConfiguration->templatesVars['displayBackOfficeHeader'] = $this->getDisplayHookRender(
+                $this->hookDispatcher->dispatchRenderingWithParameters('displayBackOfficeHeader', [])
+            );
+        }
+
+        $menuLinksCollections = new ActionsBarButtonsCollection();
+
+        $this->hookDispatcher->dispatchWithParameters(
+            'displayBackOfficeEmployeeMenu',
+            [
+                'links' => $menuLinksCollections,
+            ]
+        );
+
+        $controllerConfiguration->templatesVars['displayBackOfficeTop'] = $this->getDisplayHookRender(
+            $this->hookDispatcher->dispatchRenderingWithParameters('displayBackOfficeTop', [])
+        );
+        $controllerConfiguration->templatesVars['displayBackOfficeEmployeeMenu'] = $menuLinksCollections;
+        $controllerConfiguration->templatesVars['submit_form_ajax'] = (int) Tools::getValue('submitFormAjax');
+
+        $tabs = $this->getTabs($controllerConfiguration);
+        $currentTabLevel = 0;
+        foreach ($tabs as $tab) {
+            $currentTabLevel = isset($tab['current_level']) ? $tab['current_level'] : $currentTabLevel;
+        }
+
+        $controllerConfiguration->templatesVars['bo_query'] = Tools::safeOutput(Tools::stripslashes(Tools::getValue('bo_query')));
+        $controllerConfiguration->templatesVars['collapse_menu'] = isset($this->cookie->collapse_menu) ? (int) $this->cookie->collapse_menu : 0;
+        $controllerConfiguration->templatesVars['default_tab_link'] = $this->link->getAdminLink(Tab::getClassNameById((int) $controllerConfiguration->user->getData()->default_tab));
```

Comment:
I can't use to seeing `getUserSomething` instead of `getEmployeeSomething` ;-)

Reply:
Do you want me to rename `user` to `employee`?

Reply:
@tegbessou I'd recommend doing so but I would love to hear about it from other @PrestaShop/prestashop-maintainers too